client/.husky/pre-commit

```diff
@@ -1,9 +1,13 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
-SRC_PATTERN="client/"
+# The following lines have been commented out to disable automatic execution of the pre-commit linting and formatting tasks.
```

Comment:
Can we just remove husky now? Is it needed still?

Reply:
From what I can see, not needed anymore. I was going to suggest that we just remove it now

Reply:
Have removed it. Please see changes here: https://github.com/msupply-foundation/open-msupply/pull/6796/commits/5969e2980af3f26e056f8004ef22771b4ec6ae7b

I've dropped these libraries
- Husky
- Lint-staged